frontend/apps/reader/modules/readerthumbnail.lua

```diff
@@ -78,6 +79,10 @@ function ReaderThumbnail:addToMainMenu(menu_items)
 end
 
 function ReaderThumbnail:onShowBookMap()
+    if self.view.inverse_reading_order then
```

Comment:
We don't pass "view", so `self.ui.view.inverse_reading_order` . Otherwise it crashes (had to fix this to be able to test it ? :)

Reply:
forgot to commit it